app/assets/v2/js/grants/new.js

```diff
@@ -259,6 +262,26 @@ const init = () => {
     $('#js-token').select2();
     $("#js-token option[value='0x0000000000000000000000000000000000000000']").remove();
     $('#js-token').append("<option value='0x0000000000000000000000000000000000000000' selected='selected'>Any Token");
+
+    const categoryOptions = [
+      {value: 'security', text: 'Security'},
```

Comment:
why are you defining this in more than once place? better to add to shared.js IMHO